### ISSUE TITLE
inform user about which sims depend on the sim being translated?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/rosetta#66]
This problem goes in 2 direction. In https://github.com/phetsims/rosetta/issues/63, I described the case where a sim gets all of its sims from some other sim (e.g. Concentration is a subset of Beer's Law Lab).

The problem goes in the other direction too. If we were translating Beer's Law Lab, changes to its translation will affect Concentration. Does the user need to be aware of that? And how do we determine that dependency? (There's nothing in beers-law-lab that indicates the dependency in that direction.)

**Comment by pixelzoom** [comment]
Here's another real-world example, the suites of faradays-law sims that may someday be ported from Java.  There is one main sim, "Faraday's Electromagnetic Lab", that contains 4 screens.  Then there are 3 other sims that are various combinations of screens: "Generator", "Magnet & Compass", "Magnets & Electromagnets".  When I translate any 1 of these, I am potentially affecting the other 3.  How is the user made aware of that?

**Comment by jbphet** [comment]
I think that this would be a good design question to address in the second version of the translation utility.  The first version, which is coming up on its first round of testing by external translators, was intended to be "bare bones".  

I'll mark this issue for discussion at the next developer meeting and we can decide if the rest of the team agrees.

**Comment by pixelzoom** [comment]
Agreed that this can be addressed in a later version.  But I think putting strings from other sims under the "Sim Strings" heading in this first beta is a mistake.

**Comment by jbphet** [comment]
See https://github.com/phetsims/rosetta/issues/63#issuecomment-137538586 for decision on what to do about this in the short term.

**Comment by aaronsamuel137** [comment]
Unassigning for now. This will not be addressed in 1.0.

**Comment by jbphet** [comment]
@ariel-phet - any opinion as to whether this should be addressed as part of chipper:2.0?  While it would be an improvement, I don't think that it has been much of an issue for translators, so I'm not very inclined to put this on the list.

**Comment by ariel-phet** [comment]
@jbphet yes I would continue to punt for the moment, bigger issues for chipper 2.0
